cr/cr-core/scripts/clear_db.py

```
@@ -0,0 +1,53 @@
+Install:  pip install psycopg2-binary
+Run:      python cr-core/scripts/clear_db.py
+"""
+
+import psycopg2
+
+# ─── CONFIG ──────────────────────────────────────────────────────────────────
+DB_HOST = "localhost"
+DB_PORT = 5432
+DB_NAME = "cr_db"
+DB_USER = "postgres"
+DB_PASS = "postgres"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "DB_PASS"

(S105)